docs/technical_specs/model_libraries/gas_solid_contactors/unit_models/fixed_bed_0D.rst

```diff
@@ -0,0 +1,168 @@
+
+Construction Arguments
+----------------------
+
+The IDAES FixedBed0D model has construction arguments specific to the whole unit and to the individual regions.
+
+**Arguments that are applicable to the FixedBed0D unit as a whole are:**
+
+* dynamic - indicates whether the model will be dynamic or not, must be True (default = True).
+* has_holdup - indicates whether holdup terms are constructed or not, must be True (default = True).
+* energy_balance_type - indicates what type of energy balance should be constructed (default = useDefault):
+  
+  - EnergyBalanceType.useDefault - refer to property package for default balance type
+  - EnergyBalanceType.none - exclude energy balances
+  - EnergyBalanceType.enthalpyTotal - single enthalpy balance for material
+  - EnergyBalanceType.enthalpyPhase - enthalpy balances for each phase
+  - EnergyBalanceType.energyTotal - single energy balance for material
+  - EnergyBalanceType.energyPhase - energy balances for each phase
+
+**Arguments that are applicable to a specific region:**
+
+* gas_property_package - property package to use when calculating properties for the gas phase (default = 'useDefault'). 
+  This is provided as a Physical Parameter Block by the Flowsheet when creating the model. 
+  If a value is not provided, the ControlVolume Block will try to use the default property package if one is defined.
+* gas_property_package_args - set of arguments to be passed to the gas phase Property Blocks when they are created (default = 'None').
+* solid_property_package - property package to use when calculating properties for the solid phase (default = 'useDefault'). 
+  This is provided as a Physical Parameter Block by the Flowsheet when creating the model. 
+  If a value is not provided, the ControlVolume Block will try to use the default property package if one is defined.
+* solid_property_package_args - set of arguments to be passed to the solid phase Property Blocks when they are created (default = 'None').
+* reaction_package - reaction package to use when calculating properties for solid phase reaction (default = None). 
+  This is provided as a Reaction Parameter Block by the Flowsheet when creating the model. 
+  If a value is not provided, the ControlVolume Block will try to use the default property package if one is defined.
+* reaction_package_args - set of arguments to be passed to the Reaction Blocks when they are created (default = None).
+
+Constraints
+-----------
+
+In the following, the subscripts :math:`g` and :math:`s` refer to the gas and solid phases, respectively. 
+FixedBed0D units write the following Constraints:
+
+Reaction and Mass/Heat Transfer Constraints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Solid material holdup:
+
+.. math:: h_{mass,s,t,j} = V_{s} \rho_{mass,s,t} x_{mass,s,t,j}
+
+Solid material accumulation:
+
+.. math:: {\dot{h}}_{mass,s,t,j} = V_{s} {MW}_{s,j} {\Sigma}_{r} {r_{s,t,r} \nu_{s,j,r}}
```

Comment:
"h" is usually associated with enthalpy. Maybe replace "h with "q"? "q" is a bit more standard for solid fluxes.